src/components/Devotions/Devotions.js

```diff
@@ -2,16 +2,21 @@ import React, {useState, useEffect, useContext, Fragment} from "react";
 import ReactDOM from 'react-dom';
 import {Input, Label, Button, Row, Col} from 'reactstrap';
 import {VoiceCommandsContext} from "../../context/VoiceCommandsContext";
-// import AnalogClock from 'analog-clock-react';
 import axios from 'axios';
 import AnalogClock from "analog-clock-react";
 
+const refreshPageCommand = {
+    command: ["mirror mirror refresh page", "mirror mirror refresh devotion", "Mirror mirror give me a new verse"],
+    answer: "",
+    func: () => window.location.reload()
```

Comment:
Art, the reload() method does get the job done, but it seem like its a little over kill (will it have any side effect?).   it could be done something like this
func: () => getQuote();

const getQuote = () => {
axios.get("https://type.fit/api/quotes")
    .then(responce => {
        const quote = responce.data; 
        console.log(quote)
        setQuote(quote) ;
}

Reply:
Thank you Jeff, I was  not seeing the obvious. I was braking my head over trying to find a way to not have it log out every time.

Reply:
I think we already merged this code. Did we?